packages/block-editor/src/components/block-list/use-block-props/use-is-hovered.js

```diff
@@ -2,37 +2,23 @@
  * WordPress dependencies
  */
 import { useRefEffect } from '@wordpress/compose';
-import { useDispatch } from '@wordpress/data';
 
-/**
- * Internal dependencies
- */
-import { store as blockEditorStore } from '../../../store';
+function listener( event ) {
+	if ( event.defaultPrevented ) {
+		return;
+	}
+
+	const action = event.type === 'mouseover' ? 'add' : 'remove';
+
+	event.preventDefault();
+	event.currentTarget.classList[ action ]( 'is-hovered' );
```

Comment:
```suggestion
	event.preventDefault();
	event.currentTarget.classList.toggle( 'is-hovered', event.type === 'mouseover' );
```

The `toggle` method also supports a secondary `force` parameter that we can use to simplify this:

> If included, turns the toggle into a one way-only operation. If set to false, then token will only be removed, but not added. If set to true, then token will only be added, but not removed.
> _(See [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle#force))_

Reply:
Thanks for the suggestion, @fabiankaegy!